test/NuGet.Core.Tests/NuGet.Common.Test/PathValidatorTest.cs

```diff
@@ -48,13 +48,14 @@ public void PathValidatorTest_ValidUncSharePath(string path, string os)
         }
 
         [Theory]
-        [InlineData(@"C:", "windows")]
-        //[InlineData(@"\\server\invalid\*\","windows")] TODO: Enabled once this issue is fixed: https://github.com/NuGet/Home/issues/7588
```

Comment:
This validator is used by package sources page. It only checks Windows paths, so I removed the checks.
Also closed this TODO issue which is no longer reproducing: https://github.com/NuGet/Home/issues/7588#issuecomment-2936602184